pom.xml

```diff
@@ -279,6 +279,9 @@
 			<plugin>
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
+		                <configuration>
+                                        <classifier>exec</classifier>
```

Comment:
Just briefly checked on this: This would be incompatible for maven central downloaders 😢 
So far, our "full-blown package" has no classifier.
With this way, the "full-blown package" would get a classifier "exec". The "promregator library" would not have the classifier. So people so far just downloading the thing without any suffix will now bump into the library instead of the "full version". We need to turn this around somehow.

I think we need to further play around with the [options of `repackage`](https://docs.spring.io/spring-boot/docs/1.1.x/reference/html/howto-build.html) here.

Reply:
This is correct.  Just plainly implementing this in this way would cause issues for those depending on getting the executable when providing no classifier at all. The supposed pseudo-standard here is to have the client request the "exec" classifier, but I can absolutely see how that would be very painful given that many are already using it.

Would you be comfortable with a classifier of ```lib``` or ```library``` (or something along those lines) for pulling the dependency jar?  This would keep behavior the same for existing clients (no classifier to get executable, full-blown jar) and only those who wish to use it as a dependency would provide the above classifier...

I am open to whatever alternative is best in this case, as long as we can also reference Promregator as a dependency.